tla-bmcmt/src/main/scala/at/forsyte/apalache/tla/bmcmt/caches/ExprCache.scala

```diff
@@ -28,6 +28,7 @@ class ExprCache(val store: ExprGradeStore) extends SimpleCache[TlaEx, (TlaEx, Ex
   override def put(key: TlaEx, valueAndGrade: (TlaEx, ExprGrade.Value)): Unit = {
     valueAndGrade._2 match {
       case ExprGrade.Constant | ExprGrade.StateFree | ExprGrade.ActionFree =>
+        // TODO: Temp. disabled, re-enable once caching is tested
         super.put(key, valueAndGrade)
```

Comment:
Actually, this seems like a comment that was accidentally left in, since we're not actually disabling the cache in this PR. I'll just remove the comment.